x-pack/plugin/search-business-rules/build.gradle

```diff
@@ -21,7 +21,7 @@ dependencies {
 }
 
 // copied from CCR
-dependencyLicenses {
+tasks.named("dependencyLicenses").configure {
   ignoreSha 'x-pack-core'
```

Comment:
Is this necessary at all? We shouldn't need to exclude any ES internal dependencies?

Reply:
Since we ignore all `org.elasticsearch*`artifacts it shouldn't I guess. will remove the unrequired es related configs too